spec/girl_spec.rb

```diff
@@ -223,4 +223,32 @@
     it { expect(girl).to be_yellow }
     it { expect(girl).not_to be_pink }
   end
+
+  describe "birthday?" do
+    subject { girl.birthday?(curent_date) }
+
+    let(:curent_date) { Date.today }
```

Comment:
Rails/Date: Do not use `Date.today` without zone. Use `Time.zone.today` instead.